src/content/reference/gsctl/update-org-set-credentials.md

```diff
@@ -0,0 +1,74 @@
++++
+title = "gsctl Command Reference: update organization set-credentials"
+description = "The 'gsctl update organization set-credentials' command allows to set cloud provider credentials for an organization."
+date = "2018-09-11"
+type = "page"
+weight = 55
++++
+
+# `update organization set-credentials`
+
+Giant Swarm allows you to run clusters in your own cloud provider account/subscription. We call this by the catchy term "Bring Your Own Cloud" (BYOC). As a prerequisite, the organizaiton that should own the clusters has to be prepared with cloud privider credentials.
+
+Please refer to our detailed guides on how to prepare roles and credentials inside you AWS account or Azure subscription:
+
+- [Prepare an AWS account to run Giant Swarm tenant clusters](/guides/prepare-aws-account-for-tenant-clusters/)
+- [Prepare an Azure subscription to run Giant Swarm tenant clusters](/guides/prepare-azure-subscription-for-tenant-clusters/)
+
+gsctl provides this command to store the credentials connected to your organization. After doing this, you will want to [create a cluster](../create-cluster/) owned by the organization configured that way.
+
+**Note:** The credentials of an organization are currently immutable. Once set, you cannot modify or remove them. However you can create and delete organization at will.
+
+## Usage
+
+Regardless which is your cloud provider, you need an organization to work with. If you haven't created one for the purpose yet, this can be done in our web user interface.
```

Comment:
`Regardless which is your cloud provider` -> `Regardless of your cloud provider`